elytron-oidc-client/src/test/java/org/wildfly/extension/elytron/oidc/ElytronOidcClientSubsystemTransformerTestCase.java

```diff
@@ -0,0 +1,127 @@
+import org.jboss.as.model.test.ModelTestControllerVersion;
+import org.jboss.as.model.test.ModelTestUtils;
+import org.jboss.as.subsystem.test.AbstractSubsystemSchemaTest;
+import org.jboss.as.subsystem.test.AdditionalInitialization;
+import org.jboss.as.subsystem.test.KernelServices;
+import org.jboss.as.subsystem.test.KernelServicesBuilder;
+import org.jboss.as.version.Stability;
+import org.jboss.dmr.ModelNode;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.wildfly.extension.elytron.oidc.OidcTestCase.DefaultInitializer;
+
+@RunWith(value = Parameterized.class)
+public class ElytronOidcClientSubsystemTransformerTestCase extends AbstractSubsystemSchemaTest<ElytronOidcSubsystemSchema> {
+
+    @Parameterized.Parameters
+    public static Collection<Object[]> parameters() {
+        return List.<Object[]>of(
+                new Object[] { ModelTestControllerVersion.EAP_8_0_0, ElytronOidcClientSubsystemModel.VERSION_2_0_0.getVersion() }
+        );
+    }
+    private final ModelTestControllerVersion controllerVersion;
+    private final ModelVersion modelVersion;
+    private static final PathAddress SUBSYSTEM_ADDRESS = PathAddress.pathAddress(ModelDescriptionConstants.SUBSYSTEM, ElytronOidcExtension.SUBSYSTEM_NAME);
+
+
+    public ElytronOidcClientSubsystemTransformerTestCase(ModelTestControllerVersion controllerVersion, ModelVersion version) {
+        super(ElytronOidcExtension.SUBSYSTEM_NAME, new ElytronOidcExtension(), ElytronOidcSubsystemSchema.VERSION_2_0_PREVIEW, ElytronOidcSubsystemSchema.CURRENT.get(Stability.PREVIEW));
+        this.controllerVersion = controllerVersion;
+        this.modelVersion = version;
+    }
+
+    @Test
+    public void testTransformations() throws Exception {
+        final ModelVersion version = modelVersion;
+        KernelServices services = this.buildKernelServices(controllerVersion, version);
+
+        checkSubsystemModelTransformation(services, version, null, false);
+        ModelNode transformed = services.readTransformedModel(version);
+        Assert.assertTrue(transformed.isDefined());
+    }
+
+    @Test
+    public void testRejection() throws Exception {
+        testRejectingTransformers(ModelTestControllerVersion.EAP_8_0_0, "elytron-oidc-client-reject.xml", new FailedOperationTransformationConfig()
+                .addFailedAttribute(SUBSYSTEM_ADDRESS.append(PathElement.pathElement(ElytronOidcDescriptionConstants.SECURE_DEPLOYMENT, "wildfly-reject-deployment-with-scope.war"), PathElement.pathElement(ElytronOidcDescriptionConstants.SCOPE)),
+                        new FailedOperationTransformationConfig.NewAttributesConfig(ElytronOidcDescriptionConstants.SCOPE))
+                .addFailedAttribute(SUBSYSTEM_ADDRESS.append(PathElement.pathElement(ElytronOidcDescriptionConstants.SECURE_SERVER, "wildfly-reject-deployment-with-scope.war"), PathElement.pathElement(ElytronOidcDescriptionConstants.SCOPE)),
```

Comment:
@PrarthonaPaul This should be `wildfly-with-scope.war` instead of the "wildfly-reject-deployment-with-scope.war" r

Reply:
Thanks for catching it! 
Should be good now. 
Thanks @Skyllarr